nservicebus/messaging/databus/custom.md

```diff
@@ -12,10 +11,6 @@ redirects:
 
 Endpoints support sending and receiving large chunks of data via the [DataBus](./).
 
-It is possible to create a custom DataBus implementation by implementing the `IDataBus` interface, such as in the following minimalistic sample:
+partial: databus_content
 
-snippet: CustomDataBus
-
-To configure the endpoint to use the custom DataBus implementation it is enough to register it at endpoint configuration time, such as in the following sample:
-
-snippet: PluginCustomDataBus
\ No newline at end of file
+When customizing DataBus, it may be desirable to also [customize the DataBus serializer](/samples/databus/custom-serializer/) to use Json instead of binary serialization.
```

Comment:
may be desirable? How would I make that decision? Should this just be a related article?